src/test/java/com/spotify/faceapi/Service/TopTrackofArtistTest.java

```diff
@@ -0,0 +1,122 @@
+        Map<String, String> topArtiststracks_map1 = mapper.readValue(json_obj3.toString(), Map.class);
+
+        ResponseEntity responseEntity = new ResponseEntity(topArtiststracks_map1, HttpStatus.OK);
+        Mockito.when(restTemplate.exchange(ArgumentMatchers.anyString(),
+                ArgumentMatchers.any(HttpMethod.class),
+                ArgumentMatchers.<HttpEntity<?>> any(),
+                ArgumentMatchers.<Class<String>> any())).thenReturn(responseEntity);
+
+        LinkedHashMap res = (LinkedHashMap) topTrackofArtist.getTopTracksofArtist("AZT1010ZZ7H2","1mYsTxnqsietFxj1OgoGbG");
+        Assert.assertEquals(topArtiststracks_map1,res);
+    }
+
+    @Test
+    public void getTopTracksofArtist_NoContent() throws NoAccountException, ForbiddenException, NoArtistsException, BadRequestException, NotFoundException, UnAuthorizedException {
+        LinkedHashMap<String, ArrayList> inputMap = new LinkedHashMap<>();
+        ResponseEntity responseEntity = new ResponseEntity(inputMap, HttpStatus.NO_CONTENT);
+        Mockito.when(restTemplate.exchange(ArgumentMatchers.anyString(),
+                ArgumentMatchers.any(HttpMethod.class),
+                ArgumentMatchers.<HttpEntity<?>> any(),
+                ArgumentMatchers.<Class<String>> any())).thenReturn(responseEntity);
+        String Message = null;
+        try{
+            topTrackofArtist.getTopTracksofArtist("AZT1010ZZ7H2","1mYsTxnqsietFxj1OgoGbG");
+        }catch (Exception e){
+            Message = e.getMessage();
+        }
+        Assert.assertEquals(AppConstants.NoTrackException_Msg, Message);
+    }
+
+    @Test
+    public void getTopTracksofArtist_Forbidden() {
+        Mockito.when(restTemplate.exchange(ArgumentMatchers.anyString(),
+                ArgumentMatchers.any(HttpMethod.class),
+                ArgumentMatchers.<HttpEntity<?>> any(),
+                ArgumentMatchers.<Class<String>> any())).thenThrow(HttpClientErrorException.Forbidden.class);
+        String Message = null;
+        try{
+            topTrackofArtist.getTopTracksofArtist("AZT1010ZZ7H2","1mYsTxnqsietFxj1OgoGbG");
+        }catch (Exception e){
+            Message = e.getMessage();
+        }
+        Assert.assertEquals(AppConstants.ForbiddenException_Msg, Message);
+    }
+
+    @Test
+    public void getTopTracksofArtist_BadRequest() {
+        Mockito.when(restTemplate.exchange(ArgumentMatchers.anyString(),
+                ArgumentMatchers.any(HttpMethod.class),
+                ArgumentMatchers.<HttpEntity<?>> any(),
+                ArgumentMatchers.<Class<String>> any())).thenThrow(HttpClientErrorException.BadRequest.class);
+        String Message = null;
+        try{
+            topTrackofArtist.getTopTracksofArtist("AZT1010ZZ7H2","1mYsTxnqsietFxj1OgoGbG");
+        }catch (Exception e){
+            Message = e.getMessage();
+        }
```

Comment:
Line 115-119 have conventional formatting, but here it's different.  Inconsistent.  Linter/formatter?